### PR TITLE
Setup json and different fetches for pages 

### PR DESCRIPTION
**Pull request by Phil9219**
Maybe I will change it the way it will not say the id in the address, but the name of the run. 
What you think?

### CODE REVIEW

File: client/src/components/Page/ChooseYourRunPage.js [hunk 1]
```diff
@@ -23,37 +24,33 @@ const ScrollContainer = styled.div`
...
-            ></RunCard>
-          </Link>
+          {runs?.map((run) => (
+            <Link key={run.id} to={`/runs/${run.id}`}>
+              <RunCard
+                date={run.date}
+                time={run.time}
+                distance={run.distance}
+                isFavorite={true}
+                runName={run.runName}
```
Comment (arassaria) [inline client/src/components/Page/ChooseYourRunPage.js]:
```suggestion
                isFavorite={true}
                {...run}
```

you could simplify your code a bit 😄

File: client/src/components/Page/CreatedRunPage.js [hunk 2]
```diff
@@ -1,24 +1,39 @@
...
+
         <BigRunCard
-          date={"44.44.444"}
-          time={"14:44"}
-          distance={"10"}
+          key={run.id}
+          id={run.id}
+          date={run.date}
+          time={run.time}
+          distance={run.distance}
```
Comment (arassaria) [inline client/src/components/Page/CreatedRunPage.js]:
```suggestion
          {...run}
```
same as above 😄

File: client/src/components/Page/RunsDonePage.js [hunk 3]
```diff
@@ -21,34 +22,32 @@ const ScrollContainer = styled.div`
...
-            ></RunCard>
-          </Link>
+          {runs?.map((run) => (
+            <Link key={run.id} to={`/runs/${run.id}`}>
+              <RunCard
+                date={run.date}
+                time={run.time}
+                distance={run.distance}
+                isFavorite={true}
+                runName={run.runName}
```
Comment (arassaria) [inline client/src/components/Page/RunsDonePage.js]:
```suggestion
                isFavorite={true}
                {...run}
```

and again a simplification 😃

**Review by arassaria — COMMENTED** [review]
LGTM 🚀

**Review by mhstrkmp — APPROVED** [review]
lgtm 🚀 
nice and clean

File: client/src/components/Page/RunsDonePage.js (hunk 3 above)
Comment (Phil9219) [reply client/src/components/Page/RunsDonePage.js]: Thanks 👍

**Pull request merged**